### DISCUSSION TITLE
Create a community maintained wiki page for scipts

### DISCUSSION THREAD

**Discussion by RayZ0rr** [i3/i3#4885] [Ideas]
Fzf does have a successful [page like this](https://github.com/junegunn/fzf/wiki/Examples)

[So does Rofi](https://github.com/davatorium/rofi/wiki/User-scripts)

It would be cool if i3 also maintains something similar with a similar disclaimer that its community maintained and therefore not part of anything official.

**Comment by Airblader** [comment]
#4509

**Reply by RayZ0rr** [reply]
Yes, I have seen it and commented there. But see the links I posted in the first post. I was thinking something similar to that because it would be easier for new user and community as a whole to find it.

**Reply by Airblader** [reply]
All of this has been discussed sufficiently in the linked Discussion. I don't see anything new being mentioned here. We'll talk how i3 can officially incorporate some community-maintained list once a community-maintained list exists and has proven to remain maintained and useful.

**Reply by RayZ0rr** [reply]
I think you are misunderstanding it. Community driven page is not official at all. Its completely community-driven with the disclaimer `use at your own risk. Not part of official`.
If you look at the fzf and rofi links I have given, you'll understand it better.

**Reply by Airblader** [reply]
You're free to maintain any list you want. If you want the i3 project to link to it, that puts a burden on us to make sure we're linking to a resource that is useful and maintained. It's a burden we put onto ourselves, but it exists nonetheless.

This has been discussed at lengths in #4509. If you want to continue the discussion, please do it there instead of opening a new thread for it.

**Reply by RayZ0rr** [reply]
Ok. closing remark:

The links I posted in the first post for `fzf` and `rofi`, The official maintainers haven't added anything for maintenance. Because there is no obligation for them to add anything on their part. It's not part of official and its community-driven. No burden on the maintainers !! :smile:
